website/src/data/components/stream/props.js

```diff
@@ -233,6 +233,18 @@ const props = [
         controlType: 'inheritedColor',
         group: 'Dots',
     },
+    {
+        key: 'legendLabel',
+        help: 'Legend label accessor',
+        description: `
+            Define how to access the legend label of each datum,
+            by default, nivo will look for the \`id\` property.
+        `,
+        type: 'string | (datum: RawDatum): string',
```

Comment:
I don't think the type of `datum` is correct as [we just pass the `id`](https://github.com/plouc/nivo/pull/1562/files#diff-44b3698944ddca2af8ad0c22a159233c92c88739e99d018b7a9b9f09f345dfb6R107) if I understand correctly, maybe we could use an inlined type such as `(datum: { id: string }): string`?